frontends/precinct-scanner/src/screens/poll_worker_screen.test.tsx

```diff
@@ -147,3 +124,43 @@ describe('shows Export Results button only when polls are closed and more than 0
     expect(screen.queryByText(exportButtonText)).toBeTruthy();
   });
 });
+
+describe('shows Livecheck button only when enabled', () => {
+  test('enable livecheck', async () => {
+    // eslint-disable-next-line @typescript-eslint/require-await
+    await act(async () => {
+      renderScreen({
+        scannedBallotCount: 5,
+        isPollsOpen: true,
+        enableLivecheck: true,
```

Comment:
are we never testing the default behavior of calling isLiveCheckEnabled?

Reply:
good callout. I thought about this some more and decided that adding `enableLiveCheck` as a prop to the poll worker screen kind of overcomplicates the feature flag approach, so I took that out and instead mocked the `isLiveCheckEnabled` call.